Ventilators/R4/files/VentilatorsPrepopulation-0.1.0.cql

```diff
@@ -68,10 +25,8 @@ define DeviceRequestHcpcsCoding: singleton from (
   ((cast device_request.code as CodeableConcept).coding) coding
     where coding.system.value = 'https://bluebutton.cms.gov/resources/codesystem/hcpcs')
 
-define DeviceRequestDescription: 'HCPCS ' + "DeviceRequestHcpcsCoding".code.value + ' - ' + "DeviceRequestHcpcsCoding".display.value
-
-define VentilatorDeviceRequested:
-  if "DeviceRequestHcpcsCoding".code.value in {'E0465', 'E0466'} then "DeviceRequestHcpcsCoding".code.value
+define DeviceRequested:
+  if "DeviceRequestHcpcsCoding".code in "Devices" then "DeviceRequestHcpcsCoding".code.value
```

Comment:
Matching above suggestion with value set change. But the test of this line this changes the result that will end up in the questionnaire from a string that has the system, code and display into just the code. Is this an intentional change?

```suggestion
  if "DeviceRequestHcpcsCoding".code in "Ventilator Device" then "DeviceRequestHcpcsCoding".code.value
```